src/app/Components/MyPatients/my-patients.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MyPatients } from 'src/app/Models/MyPatients';
+import { merge, of as observableOf } from 'rxjs';
+import { startWith, switchMap, map, catchError } from 'rxjs/operators';
+import { DoctorsService } from 'src/app/Services/doctors.service';
+import { PatientInfo } from 'src/app/Models/PatientInfo';
+import { MatPaginator, MatSort } from '@angular/material';
+
+export const DEFAULT_AMOUNT_OF_PATIENTS_ON_PAGE = 15;
+
+@Component({
+  selector: 'app-my-patients',
+  templateUrl: './my-patients.component.html',
+  styleUrls: ['./my-patients.component.scss']
+})
+export class MyPatientsComponent implements OnInit {
+  displayedColumns = ['select', 'firstName', 'secondName', 'thirdName', 'email'];
+  dataSource: PatientInfo[] = [];
+  isLoadingResults = true;
+  isRateLimitReached = false;
```

Comment:
There are no use of isRateLimitReached. You can use it for showing that server caught error and user need to reload page or just remove it.